src/store/mutations/mutations-model.ts

```diff
@@ -135,10 +140,22 @@ export const mutateFilingId = (state: StateIF, filingId: number) => {
 }
 
 export const mutateRules = (state: StateIF, rules: any) => {
-  // state.stateModel.rules = rules // *** FUTURE: implement this
+  state.stateModel.createRulesStep = rules
   if (!state.stateModel.ignoreChanges) mutateHaveChanges(state, true)
 }
 
+export const mutateRulesConfirmed = (state: StateIF, rulesConfirmed: boolean) => {
```

Comment:
I don't remember "rules confirmed" used anywhere. Did I miss it or is it unused?

Reply:
`rulesConfirmed` is used in `UploadRules.vue`

```
<v-checkbox
  class="chk-rules"
  id="chk-confirm-rules"
  v-model="rulesConfirmed"
  label="I confirm the following items are included as required in the Rules of the Association:"
  @change="onRulesConfirmedChange()"
/>
```

Reply:
Please confirm with BAs whether we can save that the rules were confirmed, or whether they should default to False every time we reload the draft (like the Certify checkbox).

Reply:
> `rulesConfirmed` is used in `UploadRules.vue`

Oh, I didn't see a getter for it.

Reply:
confirmed with Sienna that this will need to be reset on reload of draft.  will update to mimic what certify check box does.

Reply:
updated such that rulesConfirmed is reset each time draft filing is reloaded.

Reply:
Oh I see... You have a getter and setter for the object (createRulesStep), and also individual setters for the object properties (rulesConfirmed, rulesDoc, docKey). I find that confusing.

Reply:
will take another look and see if i can simplify things.  any specific component that does this well already i can reference?

Reply:
The closest I can think of is [this](https://github.com/bcgov/business-edit-ui/blob/1dfaa19b0c4257969c72b63abe69bddca6267458/src/views/Alteration.vue#L367), where there is only one getter and one setter, and whenever one of the fields needs to be updated, the whole object is updated.

In your case, I think you would have a few places to update to implement it this way.

Which do you think it less confusing?

Reply:
I think the approach [referenced](https://github.com/bcgov/business-edit-ui/blob/1dfaa19b0c4257969c72b63abe69bddca6267458/src/views/Alteration.vue#L367) is easier to understand.  I've the code to update the `rules` state model this way and also tried to minimize the variable usage among other clean up things.